LondonBritishLibrary/orient/BLorient12513.xml

```diff
@@ -0,0 +1,614 @@
+                                            </msItem>
+                                            <msItem xml:id="ms_i2.4.34.2">
+                                                <locus from="145vc"/>
+                                                <title ref="LIT3171Commen#Tagsas34"/>
+                                            </msItem>
+                                        </msItem>
+                                    </msItem>
+                                    <colophon xml:id="coloph1">
+                                        <locus target="#147rc"/>
+                                        <note>The date given according to the Era of the martyrs 
+                                            (<date type="translation" notBefore="1522" notAfter="1523" when-custom="1239" calendar="diocletian">1239</date>) 
+                                            is the only one, which is correct. The date according to the Era of the World (7005) must be corrected to 7015 and 
+                                            the date according to the Ethiopian Era (1005) to 1515. The last mistake occurs also in <ref type="mss" corresp="BLorient737"/>.</note>
+                                        <q xml:lang="gez">ወይእዜኒ፡ ተተርጐመት፡ እምልሳነ፡ ዓረቢ፡ ለግዕዝ፡ በአፈ፡ እንባቆም፡ ወሚካኤል፡ 
+                                            በ፸፻ወ<corr resp="PRS8999Strelcyn">፲</corr>፭ዓመተ፡ ምሕረት። 
+                                            ወእምልደተ፡ ክርስቶስ፡ መድኃኒነ፡ በ፲፻<corr resp="PRS8999Strelcyn">፭፻ወ፲</corr>ወ፭ዓመት፡ 
+                                            ወእምዓመተ፡ ሰማዕታትኒ፡ በ<date type="translation" notBefore="1522" notAfter="1523" when-custom="1239" 
+                                                calendar="diocletian">፲፻ወ፪፻፴ወ፱ዓመት።</date> 
+                                            ወፍጻሜሃኒ፡ ኮነ፡ በዕለተ፡ ሰኑይ፡ አመ፡ ሠሉሱ፡ ለወርኃ፡ ነሐሴ።</q>
+                                    </colophon>
+                                </msItem>
+                            </msContents>
+                            <physDesc>
+                                <objectDesc form="Codex">
+                                    <supportDesc>
+                                        <support>
+                                            <material key="parchment"/>
+                                        </support>
+                                        <extent>
+                                            <measure unit="leaf">1+147+1</measure>
+                                            <measure unit="page" type="blank">8</measure><locus target="#Iv #4r #4v #5r #5v #10r #147v #IIv"/>
+                                            <dimensions type="outer" unit="mm">
+                                                <height>285</height>
+                                                <width>245</width>
+                                            </dimensions>
+                                        </extent>
+                                        <foliation>Quire numbers.</foliation>
```

Comment:
See Guidelines https://betamasaheft.eu/Guidelines/?id=collationDesc
`The presence, absence and further properties of quire marks (Ethiopic numbers assigned by the manuscript-makers to quires, to keep their sequence) such as location, structure, decoration or scribal hand can be described in the <signatures>[↗](https://betamasaheft.eu/Guidelines/?id=signatures) element in <collation>[↗](https://betamasaheft.eu/Guidelines/?id=collation), summarily or one by one`